index.html

```diff
@@ -9,8 +9,14 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <div class="content">
+  <div class="title-section">
     <h2 class="title">Front-end Tools</h2>
   </div>
+  <div class="tech-block">
+    <img src="assets/react_icon.svg" height=100px weight=100px/></span>
```

Comment:
do not put height and weight in the html.. Set this values on the .scss file
Also you have a spare </span>